index.html

```diff
@@ -433,11 +433,8 @@ <h2 id="projects">Projects</h2>
               command line git hook management tool.
             </li>
             <li>
-              <a href="https://github.com/icefox/git-hooks"
```

Comment:
This link was returning a 404. Looks like this user no longer has any visible repositories.